DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/compute_units/README.md

```diff
@@ -0,0 +1,151 @@
+
+# Using Compute Units To Duplicate Kernels
+This FPGA tutorial showcases a design pattern that allows you to make multiple copies of a kernel, called compute units.
+
+***Documentation***: The [oneAPI DPC++ FPGA Optimization Guide](https://software.intel.com/content/www/us/en/develop/documentation/oneapi-fpga-optimization-guide)  provides comprehensive instructions for targeting FPGAs through DPC++. The [oneAPI Programming Guide](https://software.intel.com/en-us/oneapi-programming-guide) is a general resource for target-independent DPC++ programming. 
+
+| Optimized for                     | Description
+---                                 |---
+| OS                                | Linux* Ubuntu* 18.04; Windows* 10
+| Hardware                          | Intel® Programmable Acceleration Card (PAC) with Intel Arria® 10 GX FPGA; <br> Intel® Programmable Acceleration Card (PAC) with Intel Stratix® 10 SX FPGA
+| Software                          | Intel® oneAPI DPC++ Compiler (Beta) <br> Intel® FPGA Add-On for oneAPI Base Toolkit 
+| What you will learn               | A design pattern to generate multiple compute units in DPC++ using template metaprogramming
+| Time to complete                  | 15 minutes
+
+_Notice: Limited support in Windows*; compiling for FPGA hardware is not supported in Windows*_
+
+## Purpose
+The performance of a design can sometimes by increased by making multiple copies of kernels, to make full use of the FPGA resources. Each kernel copy is called a compute unit.  
```

Comment:
Though the performance consideration is not listed in "what you will learn"/"key concepts" right now, might be nice to show the performance gain in this tutorial from not using compute units -> using N compute units if applicable (doesn't necessarily need to be in this revision/commit).

Reply:
Thank you for the suggestion, I will keep this in mind for future versions of the tutorial.

Reply:
If you make such a change in a future revision, we should apply it across all similar tutorials. This tutorial is structurally similar to the loop_unroll FPGA feature tutorial, which entails a similar area/throughput tradeoff.